Vagrantfile

```diff
@@ -1,6 +1,20 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+# Define the number of worker nodes
+n_workers = 2
+
+# Run script to create inventory file
+File.open("inventory.ini", "w") do |file|
```

Comment:
Why is it a .ini file now instead of a .cfg file?

Reply:
That worked haha, and ini is also an accepted format so don't think it should matter

Reply:
Haha okay good enough